ui/ui-components/pages/model/[modelId]/source/[sourceId]/multipleProperties.tsx

```diff
@@ -148,9 +143,10 @@ export default function Page(props) {
       );
       if (response.property) {
         successHandler.set({ message: "Property Created!" });
-        await loadProperties();
+        loadProperties();
+      } else {
+        setLoading(false);
```

Comment:
This prevents a possible update of an un-mounted componant

Reply:
I'm reading that upon successful creation, `loading` would still be set to `true`- wouldn't this prevent creating additional properties?

Reply:
Nope! because this is a component per-row.  Once we are done loading, the whole table reloads with the updated properties.  So, we keep the button disabled until then